source/globalCommands.py

```diff
@@ -3284,6 +3286,38 @@ def script_braille_cycleShowMessages(self, gesture: inputCore.InputGesture) -> N
 		msg = _("Braille show messages %s") % ShowMessages(newValue).displayString
 		ui.message(msg)
 
+	@script(
+		# Translators: Input help mode message for cycle through braille show selection command.
+		description=_("Cycle through the braille show selection states"),
+		category=SCRCAT_BRAILLE
+	)
+	def script_braille_cycleShowSelection(self, gesture: inputCore.InputGesture) -> None:
+		"""Set next state of braille show selection and reports it with ui.message."""
+		featureFlag: FeatureFlag = config.conf["braille"]["showSelection"]
+		boolFlag: BoolFlag = featureFlag.enumClassType
+		values = [x.value for x in boolFlag]
+		currentValue = featureFlag.value.value
+		nextValueIndex = (currentValue % len(values)) + 1
+		nextName: str = boolFlag(nextValueIndex).name
+		config.conf["braille"]["showSelection"] = nextName
+		featureFlag = config.conf["braille"]["showSelection"]
+		if featureFlag.isDefault():
+			displayString = featureFlag.behaviorOfDefault.displayString
+			# Translators: Used when reporting braille show selection feature flag
+			# default behavior.

```

Comment:
Translators may not be familiar with the concept of feature flag which targets developers.

```suggestion
			# Translators: Used when reporting braille show selection state
			# (default behavior).
```